.github/workflows/apolo-release.yml

```
@@ -0,0 +1,35 @@
+name: CI
+
+on:
+  push:
+    branches: [apolo]
+    tags: ["v*"]
+  pull_request:
+    branches: [apolo]
+
+jobs:
+  release:
+    name: Release image
+    runs-on: ubuntu-latest
+    if: github.event_name == 'push' && (startsWith(github.ref, 'refs/tags/v') || github.ref == 'refs/heads/apolo')
+    steps:
+      - name: Checkout commit
+        uses: actions/checkout@v4
+      - name: Login to ghcr.io
+        uses: docker/login-action@v3.1.0
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ github.token }}
+      - name: Build Docker image
+        run: make build-image
+      - name: Push image
+        run: |
+          export IMAGE_TAG=development
+          make push-image
+      - name: Build StableStudio
+        run: make build-stablestudio-image
+      - name: Push image
+        run: |
+          export IMAGE_TAG=development
+          make push-stablestudio-image
```

let's add also release steps to release `latest` image:

```suggestion
          make push-stablestudio-image
     - name: Release images
       if: github.event_name == 'push' && (startsWith(github.ref, 'refs/tags/v')
       run: |
         make push-image
         make push-stablestudio-image
```